src/Components/Payment/SavedCreditCards.tsx

```diff
@@ -151,6 +151,7 @@ export class CreditCard extends React.Component<
 
```

Comment:
I can't comment on the exact line, but it looks like you'll need to make some changes above (line 136/137 still reference `__id`)

Similarly, in the `deleteCreditCard` method you may need to update the `id`.

If you can pull up storybooks (or link to your local force) you should be able to test these cases!

Reply:
Whoops my bad